concepts/Common/Security Considerations/30 Export Vulnerabilities/05 CSV Injection.md

```diff
@@ -0,0 +1,191 @@
+If you export data from DataGrid or PivotGrid in CSV format, you should take a thread of a CSV Injection Attack (also known as a formula injection attack). Such attacks involve an injection of a malicious character sequence that is interpreted as a formula and executed within a computer network. Cell values that start with the **=**, **+**, **-**, and **@** characters can initiate an injection attack.
+
+When executed, malicious code in a formula can tamper with user data or provide unauthorized access to data and internal resources.
+
+You can encode CSV files to prevent execution of potentially harmful code in them. Pass the [encodeExecutableContent](/Documentation/ApiReference/Common/Object_Structures/ExcelExportDataGridProps/#encodeExecutableContent) option as an argument of the configuration object of the [exportDataGrid](/Documentation/ApiReference/Common/Utils/excelExporter/#exportDataGridoptions) or [exportPivotGrid](/Documentation/ApiReference/Common/Utils/excelExporter/#exportPivotGridoptions) function.
```

Comment:
```suggestion
You can encode CSV files to prevent execution of potentially harmful code within them. Pass the [encodeExecutableContent](/Documentation/ApiReference/Common/Object_Structures/ExcelExportDataGridProps/#encodeExecutableContent) option as an argument of the configuration object of the [exportDataGrid](/Documentation/ApiReference/Common/Utils/excelExporter/#exportDataGridoptions) or [exportPivotGrid](/Documentation/ApiReference/Common/Utils/excelExporter/#exportPivotGridoptions) function.
```